charts/warpstream-agent/README.md

```diff
@@ -22,57 +70,116 @@ helm upgrade --install warpstream-agent warpstream/warpstream-agent \
+
+```yaml
+config:
+  bucketURL: <WARPSTREAM_BUCKET_URL>
+  apiKey: <WARPSTREAM_AGENT_APIKEY>
+  virtualClusterID: <WARPSTREAM_VIRTUAL_CLUSTER_ID>
+  region: <WARPSTREAM_CLUSTER_REGION>
+```
+
+3. Install or upgrade the WarpStream Helm chart using your custom yaml file:
+
+```shell
+helm upgrade --install warpstream-agent warpstream/warpstream-agent -f warpstream-values.yaml
+```
+
+### Upgrading
 
-### Upgrade an existing release
+To upgrade the deployment:
 
 ```shell
 helm repo update
-helm upgrade  warpstream-agent warpstream/warpstream-agent --reuse-values
+helm upgrade warpstream-agent warpstream/warpstream-agent --reuse-values
 ```
 
-### Customize
+### Uninstalling the Chart
 
-See [`values.yaml`](./values.yaml).
+To uninstall/delete the deployment:
 
-### Install from source
+```shell
+helm uninstall warpstream-agent
+```
 
-``` shell
-git clone https://github.com/warpstreamlabs/charts
-cd charts/warpstream-agent
-helm upgrade --install warpstream-agent . \
-    --set config.bucketURL="$YOUR_OBJECT_STORE" \
-    --set config.apiKey="$YOUR_API_KEY" \
-    --set config.region="$YOUR_CONTROL_PLANE_REGION"
-    --set config.virtualClusterID="$YOUR_VIRTUAL_CLUSTER_ID"
+This command removes all the Kubernetes components associated with the chart and deletes the release.
+
+
+## Other Deployment Options
+
+### LoadBalancer Service
+The stateless nature of the WarpStream agents allows them to be deployed behind a network load balancer using `type: LoadBalancer` in your Service configuration. See the [Kubernetes Service documentation](https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer) for details.
+
+Update your Helm chart configuration with these parameters:
+
+```yaml
+service:
+  type: LoadBalancer
+
+extraEnv: 
+  - name: WARPSTREAM_DISCOVERY_KAFKA_HOSTNAME_OVERRIDE
+    value: <LOAD_BALANCER_DNS HOSTNAME>
 ```
 
-## Development
+Follow your cloud provider's load balancer documentation to complete the configuration.
+
+**Note:** If your load balancer is exposed to the internet, enabling authentication on the WarpStream agents is highly recommended. See [Authentication](https://docs.warpstream.com/warpstream/byoc/advanced-agent-deployment-options/enable-agent-auth) for more details, including how to create the client credentials in the WarpStream console.
 
-### Testing locally
+Enable authentication by setting the `WARPSTREAM_REQUIRE_AUTHENTICATION` variable:
 
-Bootstrap a Kubernetes cluster locally, for example with [kind][].
+```yaml
+service:
+  type: LoadBalancer
+
+extraEnv: 
+  - name: WARPSTREAM_DISCOVERY_KAFKA_HOSTNAME_OVERRIDE
+    value: <LOAD_BALANCER_DNS HOSTNAME>
+  - name: WARPSTREAM_REQUIRE_AUTHENTICATION
+    value: "true"
+```
 
+### Playground Mode
+Use playground mode to easily test WarpStream without needing to first create a WarpStream account. See WarpStream's ["Hello World"](https://docs.warpstream.com/warpstream/getting-started/hello-world-using-kafka) to learn more.
```

Comment:
You should add a warning somewhere that everytime a pod restarts in this mode all data will be lost and it will be effectively a brand new cluster